sopel/trigger.py

```diff
@@ -170,6 +181,11 @@ def __init__(self, own_nick, line):
                 self.tags['intent'] = intent
                 self.args[-1] = message or ''
 
+            # Search URLs if necessary, and after CTCP parsing
+            if ' :' in line:
```

Comment:
Would it not make more sense to search URLs only if the text parameter itself contains `:`?

Also, I don't believe it's strictly required for the final parameter of a command to always have a `:` prefix, e.g. if the value doesn't contain spaces, so it might not always be safe to use that as a delimiter.

Reply:
Oh... so you could have a PRIVMSG without the ` :` because the text doesn't contain any space? That's weird AF.

Reply:
Welp I'm not going to be smart with that `.urls`, it's fine.

Reply:
Fixed.

Reply:
Servers will _probably_ normalize what gets forwarded to other clients, but it's not guaranteed.